tests/js/parsing/windows_weird_cr.js

```diff
@@ -0,0 +1,3 @@
+// This is very weird windows file that has just \r to delimit newlines
+// instead of the usual \r\n under windows.
+if (foo)
{
x = 1
x = 2;
}
```

Comment:
see how github itself got confused and even swallow the carriate returns characters ...

Reply:
Amazing